README.md

```diff
@@ -19,6 +19,8 @@ Every argument is optional.
 | `package`            | The package whose API to check for semver (in Package Id Specification format, see https://doc.rust-lang.org/cargo/reference/pkgid-spec.html for reference). If not set, all packages defined in the Cargo.toml file are processed. | |
 | `manifest-path`      | Path to Cargo.toml of crate or workspace to check. If not specified, the action assumes the manifest is under the default [`GITHUB_WORKSPACE`](https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables) path. | |
 | `verbose`            | Enables verbose output of `cargo-semver-checks`. | `false` |
+| `rust-toolchain`     | Rust toolchain name to use, e.g. `stable`, `nightly` or `1.32.0`. | `stable` |
```

Comment:
```suggestion
| `rust-toolchain`     | Rust toolchain name to use, e.g. `stable`, `nightly` or `1.68.0`. | `stable` |
```
nit: I doubt cargo-semver-checks works on 1.32, let's not give examples that don't work 😅

Reply:
Oh, definitely :-p